2_crosswalk_munge.yml

```diff
@@ -0,0 +1,21 @@
+target_default: 1_crosswalk_fetch
+
+packages:
+  - dplyr
+  - scipiper
+
+sources:
+  - 2_crosswalk_munge/src/merge_id_crosswalks.R
+
+targets:
+  2_crosswalk_munge:
+    depends:
+      - 2_crosswalk_munge/out/feature_crosswalk.rds.ind
+
+  2_crosswalk_munge/out/feature_crosswalk.rds.ind:
+    command: merge_id_crosswalks(
+      merged_ind = target_name,
+      lakes_sf_ind = '1_crosswalk_fetch/out/lakes_sf.rds.ind',
+      wqp_nhd_ind = '1_crosswalk_fetch/out/wqp_nhdLookup.rds.ind')
+  2_crosswalk_munge/out/feature_crosswalk.rds:
```

Comment:
Does this require the recently PR'd version of `scipiper`?

Reply:
No, v0.0.10 is plenty. The recently merged PR at https://github.com/USGS-R/scipiper/pull/67 just affects the putters; the getters were already 1 argument.

Reply:
but i am hoping to use the simpler gd_get option in my next PR...